app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,25 @@
+class GroupsController < ApplicationController
+  def new
+    @group = Group.new
+  end
+
+  def show
+  end
+
+  def create
+    @group = Group.new(group_params)
+    @group.add_user(current_user)
+
+    if @group.save
```

Comment:
@hrs Hey so am I going crazy. In the show page here when redirected, @group show up nillclass